src/main/kotlin/services/PersonService.kt

```diff
@@ -127,8 +128,11 @@ class PersonService(
 
 fun Person.isUser(userCode: String?) = this.user?.code == userCode
 
-fun LocalDate.isBetweenIgnoreYear(start: LocalDate, end: LocalDate): Boolean =
-    (this.withYear(start.year).isEqual(start) || this.withYear(start.year).isAfter(start)) && this.withYear(end.year).isBefore(end)
+fun LocalDate.isBetweenIgnoreYear(start: LocalDate, end: LocalDate): Boolean {
+    var nextDate = MonthDay.from(this).atYear(start.year)
```

Comment:
Is this not just LocalDate.withYear with an extra step?

Reply:
Yep, you're right. Loved the fact there is a `MonthDay` object though.